Stripe/STPPaymentCardTextField.m

```diff
@@ -48,7 +48,7 @@ @implementation STPPaymentCardTextField
 @synthesize borderColor = _borderColor;
 @synthesize borderWidth = _borderWidth;
 @synthesize cornerRadius = _cornerRadius;
-@dynamic enabled;
+@dynamic enabled, isValid, valid;
```

Comment:
I don't think you need to declare `isValid` and `valid` as dynamic – the accessors are implemented here, not provided dynamically at runtime.